pkgs/development/python-modules/matplotlib/default.nix

```diff
@@ -39,7 +39,7 @@ buildPythonPackage rec {
 
   propagatedBuildInputs =
     [ cycler dateutil numpy pyparsing tornado freetype kiwisolver
-      libpng mock pytz ]
+      libpng mock pytz pillow ]
```

Comment:
pillow needs to be added to the top of the file.

It's currently "undefined"

Reply:
Fixed, sorry. I've caught some sort of a cold, so brain isn't working right half the time :(